ui/lib/apps/Statement/components/StatementsTable.tsx

```diff
@@ -190,9 +107,9 @@ export default function StatementsTable({
   const { t } = useTranslation()
   const navigate = useNavigate()
   const [items, setItems] = useState(statements)
-  const maxs = useMax(statements)
-  const [columns, setColumns] = useState(() =>
-    tableColumns(t, concise || false, maxs, onColumnClick)
+  // const maxs = useMax(statements)
+  const [columns, setColumns] = useState(
+    tableColumns(t, concise || false, statements, onColumnClick)
```

Comment:
I think `tableColumns()` is a little expensive, so use `useState(()=> ...)` can avoid call `tableColumns()` in every render.

Reply:
Indeed! This can be optimized. Feel free to open a PR later.